crowbar_framework/app/models/backup.rb

```diff
@@ -114,10 +117,10 @@ def create_archive
   end
 
   def save_archive
-    self.name = file.original_filename.split(".").first
+    self.name = file.original_filename.remove(".tar.gz")
```

Comment:
isn't it `file.original_filename` rather than `name`? I guess we need a custom validation for the file extension